libs/designer/src/lib/core/queries/connections.ts

```diff
@@ -12,13 +13,9 @@ export interface ConnectorWithParsedSwagger {
 }
 
 export const getConnectorWithSwagger = async (connectorId: string): Promise<ConnectorWithParsedSwagger> => {
-  const { connector, swagger } = await getReactQueryClient().fetchQuery(['apiWithSwaggers', connectorId.toLowerCase()], async () => {
-    const { connector, swagger } = await ConnectionService().getConnectorAndSwagger(connectorId);
-    const parsedSwagger = await SwaggerParser.parse(swagger);
```

Comment:
this is not a time consuming operation because this is not a api call, so it might not add any change

Reply:
The main change here was that `getConnector` and `getSwagger` will use the respective query caches, instead of having a third cache for `apiWithSwaggers` with them combined